apps/pod-issuer-server/src/routes/gov.ts

```diff
@@ -0,0 +1,110 @@
+import express, { Request, Response } from "express";
+import { expressjwt, Request as JWTRequest } from "express-jwt";
+import jwt from "jsonwebtoken";
+import { POD, PODEntries } from "@pcd/pod";
+import { getGovUserByEmail, getIDPODByEmail, saveIDPOD } from "../stores/gov";
+
+const gov = express.Router();
+
+gov.post("/login", (req: Request, res: Response) => {
```

Comment:
Can the logic for login be split out into helper functions shared between the routes?

Reply:
yep